lms/templates/certificates/_accomplishment-header.html

```diff
@@ -1,11 +1,12 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
 <div class="wrapper-header">
 
     <header class="header-app" role="banner">
         <h1 class="header-app-title">
-            <a class="logo" href="${logo_url}">
-                <img class="logo-img" src="${logo_src}" alt="${_('{platform_name} Home').format(platform_name=platform_name)}" />
+            <a class="logo" href="${logo_url | n, decode.utf8 }">
+                <img class="logo-img" src="${logo_src | n, decode.utf8 }" alt="${_('{platform_name} Home').format(platform_name=platform_name)}" />
```

Comment:
We still need to apply HTML escaping to these URLs. Please remove the n filter. See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#url-context-and-escaping for more details.